.golangci.yml

```diff
@@ -66,10 +67,10 @@ linters:
   disable-all: true
   enable:
     - bodyclose
+    - copyloopvar
     - dogsled
     - dupl
     - errcheck
-    - exportloopref
```

Comment:
Commit message: https://golangci-lint.run/usage/linters/#exportloopref is broken.
Do you mean https://golangci-lint.run/usage/linters/#copyloopvar ?

Reply:
Yes I included it to highlight that it is no longer listed as an available linter on the site. I'll add the additional reference to copyloopvar and a note about exportloopref no longer being available in the commit.